docroot/themes/humsci/humsci_basic/patterns/vertical-link-card/vertical-link-card.html.twig

```diff
@@ -1,28 +1,27 @@
 {% set attributes = attributes.addClass('vertical-link-card') %}
+{%- set has_image = image|render|striptags('<drupal-render-placeholder><img>')|trim -%}
+{%- set has_description = description|render|striptags('<drupal-render-placeholder><img>')|trim -%}
 
 <div{{ attributes }}>
-  {% if button %}
-    <a href="{{ button|render|striptags|trim }}" title="{{ title|render|striptags|trim }}">
+  {% if has_image %}
+    <div class="vertical-link-card__img">
+      {{ image }}
+    </div>
   {% endif %}
 
-    {% if image %}
-      <div class="vertical-link-card__img">
-        {{ image }}
-      </div>
-    {% endif %}
+  {% if title %}
+    <h2 class="vertical-link-card__title">
+      {{ title|render|striptags|trim|raw }}
+    </h2>
+  {% endif %}
 
-    <div class="vertical-link-card__content-container">
-      {% if title %}
-        <h2 class="vertical-link-card__title">{{ title|render|striptags|trim|raw }}</h2>
-      {% endif %}
-      {% if description %}
-        <div class="vertical-link-card__description">
-          {{ description }}
-        </div>
-      {% endif %}
+  {% if has_description %}
+    <div class="vertical-link-card__description">
+      {{ description }}
     </div>
+  {% endif %}
 
   {% if button %}
-    </a>
+    <a href="{{ button|render|striptags|trim }}" aria-label="{{ title|render|striptags|trim }}" class="vertical-link-card__link"></a>
```

Comment:
hmm I think something is still off here, I am getting the external link text here instead of the url. I would take a look at the horizontal or vertical card implementation to see if there's a difference